app/services/constraints_creation_service.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class ConstraintsCreationService
+  def initialize(planning_application:, constraints_params:)
+    @planning_application = planning_application
+    @constraints_params = constraints_params
```

Comment:
What about calling this `constraints_ids` or something like that? I think `params` is a little ambiguous here.

Reply:
This actually comes in from PlanX like https://github.com/unboxed/bops/pull/1106/files#diff-b615619cfe0d1ee1441e3489b67ccaae21f57a485a460bdda1ea00152f171d13R11-R20

```ruby
ActionController::Parameters.new(
    {
      "conservation_area" => true,
       "protected_trees" => true,
       "national_park" => true,
        "listed_building" => true
     }
  )
```

Reply:
I think you'll be able to see this in swagger too for the Post request